actions/get_user.yaml

```diff
@@ -9,7 +9,7 @@ parameters:
     description: "Splunk instance if not default to search (must be in config.yaml)"
     required: false
     position: 0
-  userName:
+  user_name:
```

Comment:
This change means everyone that uses this action will need to change any workflows if they upgrade the pack - which is a breaking change.
Is it necessary to update the name, and if it is, then it probably should go to a version 3.x as it means people will need to know they need to update their workflows, and the CHANGELOG should indicate that